examples/car/booking-update-request-lock.json

```diff
@@ -1,4 +1,4 @@
 {
-  "tspId": "deadbeef-deadbeef-deadbeef-beefdead-121111-211211",
+  "tspId": "64c09cf9-cd7d-422e-9d42-8ab897bab9d5",
```

Comment:
Is this random, or is that an actual tsp id on some stage?

Reply:
This is random but the same through examples, generated by uuidgen on command line